.github/CODEOWNERS

```
@@ -1,5 +1,62 @@
-/.github/ @elastic/doc-leads
-/.github/workflows/ @elastic/docs-engineering
+# Unless a later match takes precedence, someone on the docs team must review all PRs to this repo
+*                                           @elastic/docs
 
-/serverless/ @elastic/platform-docs
-/serverless/serverless-changelog.asciidoc @elastic/docs
+# Templates, workflows, and config
+/.github/                                   @elastic/doc-leads
+/.github/workflows/                         @elastic/docs-engineering
+/acrolinx/                                  @elastic/doc-leads
+
+# The docs
+/cloud-account/                             @elastic/admin-docs
+/deploy-manage/                             @elastic/admin-docs
+/explore-analyze/                           @elastic/core-docs @elastic/experience-docs 
+/extend/                                    @elastic/core-docs
+/get-started/                               @elastic/core-docs
+
+/manage-data/                               @elastic/admin-docs
+/manage-data/ingestion/                     @elastic/ingest-docs
+
+/reference/                                 @elastic/core-docs
+/reference/data-analysis/                   @elastic/docs
+/reference/data-analysis/observability/     @elastic/experience-docs
+/reference/data-analysis/kibana/            @elastic/admin-docs @elastic/experience-docs 
```

no admin content in here (just two lonely function docs) 

```suggestion
/reference/data-analysis/kibana/            @elastic/experience-docs 
```